client/src/components/campaign/round/round.html

```diff
@@ -0,0 +1,135 @@
+  <div class="round__status {{'round__status--' + $ctrl.round.status}}"
+        layout="row" layout-align="center center">
+    <span ng-if="$ctrl.round.vote_method === 'yesno'"><md-icon>thumbs_up_down</md-icon></span>
+    <span ng-if="$ctrl.round.vote_method === 'rating'"><md-icon>star_border</md-icon></span>
+    <span ng-if="$ctrl.round.vote_method === 'ranking'"><md-icon>sort</md-icon></span>
+  </div>
+  <span class="round__line"></span>
+  <div flex>
+    <h2 class="round__name"
+        layout="row" layout-align="space-between center">
+      <div layout="column" layout-align="center start">
+        <span>{{ ::$ctrl.round.name }}</span>
+        <span class="muted">{{ ::$ctrl.round.vote_method }} · {{ $ctrl.round.status }}</span>
+      </div>
+      <md-button
+          ng-click="$ctrl.editRound()"
+          ng-disabled="$ctrl.round.status !== 'paused'">
+        <md-icon>settings</md-icon>
+        Edit Round
+      </md-button>
+    </h2>
+    <md-card class="round__description" flex>
+      <md-card-content>
+        <div layout="row" layout-align="center stretch">
+          <div class="round__directions" flex="60">
+            <h4>Deadline Date</h4>
+            <p>{{ ::$ctrl.round.deadline_date | date: 'd MMM yyyy' }} · <span class="muted">{{ ::$ctrl.round.deadline_date | fromNow }}</span></p>
+            <h4>Directions</h4>
+            <p>{{ ::$ctrl.round.directions }}<span class="muted" ng-if="!$ctrl.round.directions">No directions provided.</span></p>
+          </div>
+          <div flex></div>
+          <div class="round__settings" flex="30">
+            <h4>Round File Settings</h4>
+            <p layout="column" layout-align="start stretch">
+              <span layout="row" layout-align="space-between start"
+                  ng-repeat="(name, value) in $ctrl.round.config">
+                <span>{{ ::name | translate }}</span>
+                <span>{{ ::value | json}}</span>
+              </span>
+            </p>
+            <p layout="column" layout-align="start stretch">
+              <span layout="row" layout-align="space-between start"
+                  ng-repeat="(name, value) in $ctrl.round.details.counts">
+                <span>{{ ::("round." + name) | translate }}</span>
+                <span>{{ ::value | json}}</span>
+              </span>
+            </p>
+          </div>
+        </div>
+        
+        <div class="round__jurors"
+            layout="row" layout-align="center stretch">
+          <div class="round__directions" flex>
+            <h4>Quorum</h4>
+            <p>{{ ::$ctrl.round.quorum }} jurors per photo</p>
+            <h4>Jurors</h4>
+            <div class="round__juror-container"
+                layout="column" layout-align="center stretch"
+                ng-repeat="juror in $ctrl.round.jurors | orderBy:'username'">
+              <span class="round__juror"
+                  ng-class="juror.is_active ? 'round__juror--active' : 'round__juror--inactive'">
```

Comment:
If they are cancelled and crossed out, it should not say "N/A ... 0 out of 0" below. 

Should we just remove them from the display entirely, if they have 0 votes and were removed? What do you think, @mahmoud ?